tox.ini

```diff
@@ -11,11 +11,15 @@ deps =
 commands = py.test
 
 [pytest]
-addopts=--cov=uswds_forms --cov-report=term --cov-report=html
+addopts=--cov=uswds_forms --cov-report=term --cov-report=html --ignore=venv
 
 [testenv:bandit]
 deps = bandit
-commands = bandit -r {toxinidir}/uswds_forms -x uswds_forms/tests
+
+# The inclusion of multiple identical exclusion directories with path
+# separators for Windows + UNIX is super weird but seems like the only
+# way to make this command work on both platforms.
+commands = bandit -r {toxinidir}/uswds_forms -x uswds_forms/tests,uswds_forms\tests
```

Comment:
This is really weird--tox seems to have `{:}` as a platform-independent way to put `PATH` separators (e.g. `:`/`;`) into tox configs, but it doesn't have a way to specify path separators like `/` and `\`. Consequently the bandit command was failing on Windows but working on Unix. The only way I've found to fix this is to specify *both* versions of the same directory as exclude paths, which is weird.

Also weird is the fact that specifying the directory to recurse into does not have the same issue--I was able to add a bandit vulnerability to the source and have it find it and fail.